game-core/build.gradle

```diff
@@ -11,10 +11,10 @@ dependencies {
     implementation 'com.github.openjson:openjson:1.0.10'
     implementation 'com.googlecode.soundlibs:jlayer:1.0.1.4'
     implementation 'com.sun.mail:javax.mail:1.6.2'
-    implementation 'com.yuvimasory:orange-extensions:1.3.0'
     implementation 'commons-cli:commons-cli:1.4'
     implementation 'commons-codec:commons-codec:1.11'
     implementation 'commons-io:commons-io:2.6'
+    implementation 'org.apache.commons:commons-lang3:3.8.1'
```

Comment:
Did you have a look at the implementation of the code you're using? It might be a simple check that could be reduced into a single method we could put somewhere, so we don't have to introduce this dependency.

Reply:
Had a look myself: The library does admittedly create a really nice wrapper API.
This API however boils down to:
1. Fetching a system property
https://github.com/apache/commons-lang/blob/e0a5a4b5b63068063878fa68d2568e57d9e1f4fc/src/main/java/org/apache/commons/lang3/SystemUtils.java#L449-L450
2. parsing it into an enum
https://github.com/apache/commons-lang/blob/e0a5a4b5b63068063878fa68d2568e57d9e1f4fc/src/main/java/org/apache/commons/lang3/JavaVersion.java#L182
3. comparing the corresponding float values
https://github.com/apache/commons-lang/blob/e0a5a4b5b63068063878fa68d2568e57d9e1f4fc/src/main/java/org/apache/commons/lang3/JavaVersion.java#L141

So indeed something that could be checked in a one/two-liner instead of having to add a dependency just for this single use case. (Of course if you can point out some code that we could replace with this library as well, that would be a different story)

Reply:
d8c2787fa is the simplest implementation I could come up with.  If we need to check the JRE version at runtime in a second location, we can consider adding Apache Commons Lang at that time.  (Once we drop support for Java 8, we can use `Runtime#version()`.)